docs/core/integration-quality-scale/rules/iqs052.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "IQS052 - The documentation provides automation examples the user can use."
+---
+
+IQS052 is as follows:
+
+The documentation provides automation examples the user can use.
+
+## Reasoning
+
+To show how the integration can be used, we should provide a limited set of blueprints, containing common or useful ones.
```

Comment:
But since we want blueprints and scripts (since those are ready to be used) I have a hard time integrating this in this sentence. 

(I also don't reeaally see how dashboards and automations can be shared out of the box in a neat way)

Reply:
A YAML-example of an automation is also ok, I think.

I don't know much about dashboards. Someone else should give input on that.